modules/opp-architecture-installation.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * architecture/opp-architecture.adoc
+
+:_module-type: PROCEDURE
+[id="opp-architecture-installation_{context}"]
+= Install {product-title}
+
```

Comment:
I believe having a section about minimum resource requirements specifically for OPP would go a long way to the user experience here. It took me a lot of trial and error going through all the different products documentation so finally get to a point where I was deploying these products with enough capacity on my underlying infrastructure.

Having something that describes the absolute minimum resource requirements for OPP to install sucessfully like we see in [ODF resource requirements](https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.11/html-single/planning_your_deployment/index#resource-requirements_rhodf) would be very helpful.

Reply:
Thanks for this @ascerra. This is on the list to discuss with the engineering to get in the guide soon after the first publishing. I will include you on the ODCBUGS Jira for transparency.